### PR TITLE
TOX lint config and fixes

### PR DESCRIPTION
**Pull request by radowit**

### CODE REVIEW

File: app/manage.py [hunk 1]
```diff
@@ -17,5 +19,5 @@ def main():
     execute_from_command_line(sys.argv)
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```
Comment (lprasol) [inline app/manage.py]: What's wrong with single quotes?

File: app/meetings/migrations/0001_initial.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Generated by Django 2.2.6 on 2020-01-15 21:55
 
+import django.db.models.deletion
```
Comment (lprasol) [inline app/meetings/migrations/0001_initial.py]: Why to change the order? It was alphabetical. Isort conf needs fixing?

File: app/meetings/models.py [hunk 3]
```diff
@@ -3,24 +3,32 @@
 
 
 class Guild(models.Model):
+    """Small group for users or room during a convention."""
```
Comment (lprasol) [inline app/meetings/models.py]: a/for/of ? #suggestion

File: app/zagrajmy/settings.py [hunk 4]
```diff
@@ -37,39 +37,38 @@
...
 
 MIDDLEWARE = [
-    'django.middleware.security.SecurityMiddleware',
-    'django.contrib.sessions.middleware.SessionMiddleware',
-    'django.middleware.common.CommonMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
-    'django.contrib.auth.middleware.AuthenticationMiddleware',
-    'django.contrib.messages.middleware.MessageMiddleware',
-    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    "django.middleware.security.SecurityMiddleware",
```
Comment (lprasol) [inline app/zagrajmy/settings.py]: what's wrong with single quotes again?

File: app/manage.py (hunk 1 above)
Comment (radowit) [reply app/manage.py]: Black did this. It prefers double apostrophes.

File: app/meetings/migrations/0001_initial.py (hunk 2 above)
Comment (radowit) [reply app/meetings/migrations/0001_initial.py]: isort divides import into groups:
python-internal
third party
own
and then in the groups all imports are sorted alphabetically

**Pull request merged**